### ISSUE TITLE
Test case fail, error: No module named 'veritas.utility'

### ISSUE THREAD

**Issue by DaliaGala** [mas-veritas2/veritastool#57]
In your readme/documentation, there is no mention of the requirement to install a module "veritas". I follow the process you describe and pip install veritastool. I then try to run the next cell: 

from veritastool.util.utility import test_function_cs
test_function_cs()

I get this error:

ModuleNotFoundError                       Traceback (most recent call last)

Cell In[9], line 1
----> 1 from veritastool.util.utility import test_function_cs
      2 test_function_cs()

File /usr/local/lib/python3.10/dist-packages/veritastool/__init__.py:1
----> 1 from .model_container import ModelContainer
      3 # from os.path import dirname, basename, isfile, join
      4 # import glob
      5 # modules = glob.glob(join(dirname("__file__"), "*.py"))
      6 # __all__ = [basename(f)[:-3] for f in modules if isfile(f) and not f.endswith('__init__.py')]

File /usr/local/lib/python3.10/dist-packages/veritastool/model_container.py:3
      1 #import ModelWrapper
      2 from .custom.modelwrapper import ModelWrapper
----> 3 from veritas.utility import check_datatype, check_value, check_label
      4 import numpy as np
      5 import pandas as pd

ModuleNotFoundError: No module named 'veritas.utility'

Using pip install veritas does not solve this issue. I get the same error when I run this:
from veritastool.model.modelwrapper import ModelWrapper
from veritastool.model.model_container import ModelContainer
from veritastool.usecases.credit_scoring import CreditScoring

**Comment by mas-veritas2** [comment]
i just installed the library and ran the line "test_function_cs()". the output is as expected. 
<img width="439" alt="image" src="https://user-images.githubusercontent.com/93499422/236387593-1aae4d57-4231-44f7-940a-05701a61a093.png">
could you try to create a new Python environment and re-try? and could you share the version of veritastool installed in your environment?

**Comment by mas-veritas2** [comment]
in addtion, "veritas" is not library dependency and the line "from veritas.utility import check_datatype" cannot be found in the source codes.

**Comment by DaliaGala** [comment]
Hi there, I did everything as the page states, etc. I am still getting this error, I am attaching the screenshot. I have a separate environment, all requirements installed. I am installing veritastool using pip. Is it perhaps a pip problem? This line is somewhere in some code that installs when I use pip install veritastool.
<img width="1020" alt="Screenshot 2023-05-16 at 13 08 41" src="https://github.com/mas-veritas2/veritastool/assets/65315683/3228f7b7-8216-438a-904c-ecf243a2662b">

**Comment by DaliaGala** [comment]
The version which is installed is 0.2. 
<img width="879" alt="Screenshot 2023-05-16 at 13 12 55" src="https://github.com/mas-veritas2/veritastool/assets/65315683/84f2e3e1-4077-4bc2-a4dc-e74914cbb0de">

**Comment by DaliaGala** [comment]
OK, I solved it by:
1. Trying pip install veritastool==2.0.2, latest version
2. It said I had wrong python version
3. Changed python version to 3.10
4. Installing by pip install veritastool==2.0.2  

I still don't understand why it installed version 0.2 after I executed simple pip install veritastool? Maybe on the main page of your GH/in readme it should say to use pip install veritastool==latest version and what python to use at the top of the page to avoid these problems I had.
